.github/workflows/ci-pipeline.yaml

```diff
@@ -87,8 +87,16 @@ jobs:
   vib-verify:
     runs-on: ubuntu-latest
     needs: get-chart
-    if: ${{ needs.get-chart.outputs.result == 'ok' && contains(github.event.pull_request.labels.*.name, 'verify') }}
-    name: VIB Verify
+    # Given permormance issues of the action feature on GH's side, we need to be very restrictive in the job's triggers:
+    # -> The 'Get modified charts' job suceededs AND
+    # -> The event action is different from opened AND
+    # -> The pipeline was triggered due to a label addition and said label was the 'verify' one OR
+    # -> The pipeline was NOT triggered due to a label addition but the PR already contains the 'verify' one
+    if: |
+        needs.get-chart.outputs.result == 'ok' &&
+        github.event.action != 'opened' &&
+        github.event.action == 'labeled' && github.event.label.name == 'verify' ||
+        github.event.action != 'labeled' && contains(github.event.pull_request.labels.*.name, 'verify')    name: VIB Verify
```

Comment:
I think you are missing a line break here

Reply:
My bad, I copy&pasted it without it. Thanks!